bot.py

```diff
@@ -93,9 +93,9 @@ def start_next_row (self):
             self.expect_bell (index, bell)
 
     def start_method (self):
+        assert self.row_generator.number_of_bells == self._tower.number_of_bells, \
```

Comment:
An unrelated changes, but it the bot doesn't handle this so I think it's better to fail